crates/nu-cli/src/data/value.rs

```diff
@@ -55,6 +55,15 @@ pub fn compute_values(
                 }?;
                 Ok(UntaggedValue::Primitive(Primitive::Filesize(result)))
             }
+            (Primitive::Filesize(x), Primitive::Int(y)) => match operator {
```

Comment:
If we add the `into-int` command, we don't need this part (and probably shouldn't have it, at least for now). We'll instead just let people do the conversion to `int` manually rather than doing it automatically for them.

Reply:
Oops, I forgot to revert it.
It looks like I've caused a lot of confusion for others with this.